### DISCUSSION TITLE
NameError: Could not load Llama model from path: ./gpt4all/ggml-model-q4_0.bin

### DISCUSSION THREAD

**Discussion by akashlinux10may** [ggml-org/llama.cpp#1478] [Q&A]
help me on this

i downloaded ggml-model-q4_0.bin from
https://huggingface.co/Pi3141/alpaca-native-7B-ggml/commit/397e872bf4c83f4c642317a5bf65ce84a105786e

**Comment by okpatil4u** [comment]
Say please at least 😂

**Reply by akashlinux10may** [reply]
please :-D
